packages/office-ui-fabric-react/src/components/Persona/Persona.styles.ts

```diff
@@ -71,6 +72,7 @@ export const getStyles = (
     root: [
       classNames.root,
       normalize,
+      getFocusStyle(theme, -2),
```

Comment:
Do we want Persona to get focus styles?

Reply:
no. If you want it to be focused, wrap it in a button or anchor which should receive focus styling.